### PR TITLE
Mm/create form

### LINKED ISSUE #5: Create Forms to allow people to send email to join or ask questions
**Context**
We need to create forms to allow visitors to request to join Technolgap's private slack or ask questions. We can use [FormSpree](https://formspree.io/) without using server-rendering code.

**Problem**
We don't have our forms...yet

**Acceptance Criteria:**
* The appearance of this component's desktop version must be exact or (very) similar to the approved [Figma Mockup](https://www.figma.com/file/ciTL29R7gAaKkipH6v8QLL/Technolgap-official-Mockups?node-id=511%3A661)

> Note: you're not responsible for the Title component. It is covered in #7 but you will have to call it.

* When user clicks on the **Submit**, it should successfully send it to the Technolgap email (TBA) -> this should be handled by FormSpree
Screen Shot 2020-08-23 at 8 55 14 AM

* Each form should be in a new page

### PR DESCRIPTION
**Pull request by mukta-m**
I made the two forms! The questions/placeholder for the sign up form can be adjusted, I tried doing what the Google Form was at one point in time. I wasn't sure if we wanted a different colour when user is hovered or focused on an input/textarea, so it's in the css, but commented out.

**questions**
Screen Shot 2020-09-29 at 10 02 16 AM

**sign up**
Screen Shot 2020-09-29 at 10 01 50 AM

closes #5

### CODE REVIEW

**Comment by ginabak** [comment]
HELLO!!! 

Thanks for all your hard work 🥰🥰🥰

ONE 🚨IMPORTANT🚨 THING: 

Let's change the word from `FEMALE` to `woman`, along the lines of "identify as a woman" 

https://www.buzzfeed.com/tracyclayton/stop-calling-women-females

File: components/Form/Form.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { getFontDefinitionFromManifest } from "next/dist/next-server/server/font-utils";
+import styles from "./Form.module.scss";
+
+const Form = () => 
+
+    <div className={styles.Form}>
```
Comment (ginabak) [inline components/Form/Form.js:6]: Can we try not to use `div` but rather `section` so it's more accessible 🥰 and semantic

https://dev.to/kenbellows/stop-using-so-many-divs-an-intro-to-semantic-html-3i9i

**Comment by Monica-Vu** [comment]
Screen Shot 2020-10-10 at 6 43 06 PM

Can you add more white space on the top and bottom of the form of the question? This is what it looks like for my screen. Can you make the font bigger for the input field? I personally find it a bit hard to see. 

Also, I would like the button to change colour when it hovers. Thank you!

File: components/Form/Form.module.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
+@import "../../theme/theme";
+
+.Form {
+    list-style: none;
+    // display: flex;
```
Comment (Jo-Sundaram) [inline components/Form/Form.module.scss:5]: Just be sure to remove any unnecessary comments before merging to master! I think everything looks really good!

**Review by Jo-Sundaram — APPROVED** [review]

**Comment by Monica-Vu** [comment]
Merging

**Pull request merged**